### ISSUE TITLE
Question: What about the CDN service?

### ISSUE THREAD

**Issue by Xplouder** [flyimg/flyimg#179]
Using this service, even if we gain the flexibility to manipulate the images as we want, we lose the performance benefits that a CDN directly provide right? I mean, in the best scenario even if we store the cache in a CDN, this service will always be a proxy for it, right? Resulting in adding a layer to our request.
Thanks!

**Comment by baamenabar** [comment]
Nope, actually I have https://i.m0.cl behind a CDN myself (cloudflare), but any CDN should work just as fine.

The "caching" flyimg does is for the transformation of the image, which is stored as a file and served through an nginx server regularly. That image we serve with the proper http headers so a CDN can cache it for a year (the default expiration time we have for images), although most CDNs have a lower TTL and will hit the origin server at least once a day (Like amazon Cloudfront). 

In short Flyimg can be used as an *origin* server from where a CDN could get and cache images.

I hope this anserws your question, if not give me a bit more detail on your planned or current setup and I can come up with a deeper explanation.
Cheers!

**Comment by Xplouder** [comment]
I see, so you use a CDN on top of it, makes sense. Thnak you for the quick response.
